cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -228,6 +231,7 @@ func upgradeComponent(component string, waiter apiclient.Waiter, pathMgr StaticP
 
 	if waitForComponentRestart {
 		fmt.Println("[upgrade/staticpods] Waiting for the kubelet to restart the component")
+		fmt.Println("[upgrade/staticpods] This operation will timeout in ", UpgradeManifestTimeout)
```

Comment:
we should use `Printf()`:
`fmt.Printf("[upgrade/staticpods] This operation will timeout in %v\n", UpgradeManifestTimeout)`

Reply:
ok, but this will print the same thing as what I added...

Reply:
yes, definitely!
the kubeadm codebase uses `Printf()` in all places where an argument is printed and `Println()` if no arguments are printed.

Reply:
Got you!
So kind you are. Thank you for your help!